LICENSE

```diff
@@ -1793,3 +1620,1012 @@ ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+modification, are permitted provided that the following conditions are
+met:
+
+  o Redistributions of source code must retain the above copyright
+    notice, this list of conditions and the following disclaimer.
+
+  o Redistributions in binary form must reproduce the above copyright
+    notice, this list of conditions and the following disclaimer in
+    the documentation and/or other materials provided with the
+    distribution.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+========================================================================
+getopt.c
+
+License information for getopt.c. This file is only used on Windows
+builds of the executables in the examples folder:
+
+/*
+ * This file was copied from the following newsgroup posting:
+ *
+ * Newsgroups: mod.std.unix
+ * Subject: public domain AT&T getopt source
+ * Date: 3 Nov 85 19:34:15 GMT
+ *
+ * Here's something you've all been waiting for:  the AT&T public domain
+ * source for getopt(3).  It is the code which was given out at the 1985
+ * UNIFORUM conference in Dallas.  I obtained it by electronic mail
+ * directly from AT&T.  The people there assure me that it is indeed
+ * in the public domain.
+ */
+
+========================================================================
+OpenSSL
+
+Binaries that are linked against OpenSSL include software developed
+by the OpenSSL Project for use in the OpenSSL Toolkit.
+(http://www.openssl.org/). Please consult the OpenSSL license
+(https://www.openssl.org/source/license.html) for licensing terms.
+
+========================================================================
+GnuTLS
+
+When compiled with GnuTLS support, this software includes components
+that are licensed under the terms of the the GNU Lesser General Public
+License, version 2.1
+(https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html).
+
+========================================================================
+tinyDTLS
+
+When compiled with tinyDTLS support, this software includes components
+that are licensed under the terms of the Eclipse Distribution License 1.0
+(http://www.eclipse.org/org/documents/edl-v10.php).
+
+This project bundles 'open62541' which is available under an MPLv2.0 license,
```

Comment:
It is my understanding this is a binary inclusion.  We should be explicit about that fact as it is a weak copyleft license and its use outside of that capacity would not be okay.  I see the conditional nature of it is highlighted in the README, but we should be also update the language here.

Reply:
Yep, it is a binary-only inclusion, will update the language to reflect this.